### ISSUE TITLE
Add the ability to suppress self-boosts

### ISSUE THREAD

**Issue by kevincianfarini** [mastodon/mastodon#21632]
### Pitch

I would love the ability to suppress self-boosts in my feed either on a case-by-case _or_ global basis. I define a self-boost to be an account boosting a toot they have made previously. 

### Motivation

I try to curate my feed to maximize my signal to noise ratio. Some accounts I follow post valuable information, but also self-boost their own toots after several days have elapsed. Presumably they do this for posts which do not receive the attention they hoped to get on certain posts. 

This behavior makes my feed less usable, but I don't think it justifies suppressing _all_ boosts from a particular account or unfollowing them altogether. I currently only follow 29 people, two of which are frequent self-boosters. I am unsure if this problem is a product of the specific niche I involve myself with, or if this is a more widespread problem on mastodon. 

I am curious to hear if other people experience this issue and would be interested in this feature!

**Comment by supernovae** [comment]
Filtering boosts from people is already part of the functionality.  Self-boosting is/should be normal.  https://docs.joinmastodon.org/user/moderating/#hide-boosts

**Comment by kevincianfarini** [comment]
@supernovae I should have been more specific. I don't want to suppress _all_ boosts by people who exhibit this behavior as the docs outline. 

> If you hide boosts from someone, you won’t see their boosts in your home feed. This option only appears on users who you are currently following.

I find many of their boosts to be relevant. I don't, however, want to see a self-boosted toot which I would have otherwise already seen (because I already follow this person). Filtering self-boosts is a subset of functionality from filtering _all_ boosts from a particular user.

**Comment by n2burns** [comment]
@100000BC I think your request is off topic to this issue. However, https://github.com/mastodon/mastodon/issues/21632#issuecomment-1335673624 provides a link which might be useful to you.

**Comment by themantripleman** [comment]
Yes, the ability to filter out relentless self-boosts would be great. There are accounts that post valuable/interesting stuff but, holy crap, I don't need to see it boosted again and again and again *by the original poster* 🤦.

**Comment by timtfj** [comment]
I came here to request this feature. There are valid reasons to boost one's own toot—for example because you posted it at a time of day when most of the followers would miss it—but some people make such a habit of it that it gets in the way of reading their new stuff.

I do feel self-boosts are enough of a separate category to warrant having their own filter, independent of muting normal boosts.

**Comment by nikclayton** [comment]
You may already be using a client that supports this feature: https://pachli.app/pachli/2023/10/25/1.3-release.html#add-a-preference-to-hide-self-boosts
